index.html

```diff
@@ -1,155 +1,165 @@
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Dienebi Ombu - Portfolio</title>
-    <link rel="stylesheet" href="styles.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
-  </head>
-  <body>
-    <header>
-      <nav class="toolbar">
-        <a class="logo" href="index.html">ebidien</a>
-        <ul class="menu-items">
-          <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
-          <li><a href="#about" class="menu-links">About</a></li>
-          <li><a href="#contact" class="menu-links">Contact</a></li>
-        </ul>
-        <img class="main-menu" src="menu-button.svg" alt="Main Menu" />
-      </nav>
-    </header>
-    <section class="headline">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Dienebi Ombu - Portfolio</title>
+  <link rel="stylesheet" href="styles.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+
+<body>
+  <header>
+    <nav class="toolbar">
+      <a class="logo" href="index.html">ebidien</a>
+      <ul class="menu-items">
+        <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
+        <li><a href="#about" class="menu-links">About</a></li>
+        <li><a href="#contact" class="menu-links">Contact</a></li>
+      </ul>
+      <img class="main-menu" src="menu-button.svg" alt="Main Menu" />
+    </nav>
+  </header>
+  <section class="headline">
+    <div>
+      <div class="intro-title">
+        <h1>I'm Dienebi. Glad to see you!</h1>
+      </div>
+      <div class="intro-body">
+        <p>
+          I'm a software developer! I can help you build a product, feature or website. Look through some of my work and
+          experience! If you like
+          what you see and have a project you need coded, don't hesitate to contact me.
+        </p>
```

Comment:
- [ ] The headline paragraph is overcrossing its parent element. Kindly ensure that the width of the paragraph is flexible for every desktop screen size ☺️ . By fixing this, you will be making the project more responsive.

- Your design 😎 
![para my](https://user-images.githubusercontent.com/99394374/213885288-b18d781b-5d4c-4c44-9605-89b3a74d1638.PNG)